### DISCUSSION TITLE
Using regexps to match multiple windows in rules

### DISCUSSION THREAD

**Discussion by bustinbung** [asmvik/yabai#1935] [Q&A] [answered]
Reading the documentation says that I can use regexps to match window properties. I'm trying to get yabai to float all Adobe applications, so I made a rule with a regex to match the word "Adobe" and everything following, but I haven't gotten the results that I've wanted. I checked my regexp using `grep -E`, and it worked there, so I'm not sure why it isn't working. I want to do it this way to avoid having to manually declare every app that I want to float.

```sh
yabai -m rule --add app="Adobe.*"
```

**Comment by nocksock** [comment]
Your rule works for me:

```sh
yabai -m rule --add app="Adobe.*" manage=off layer=below
```

Tested with Photoshop and Lightroom with yabai v6.0.0

**Reply by bustinbung** [reply, marked as answer]
Had some strange thing in my config that was breaking everything, but got it fixed!
